docs/how_tos/theming.md

```diff
@@ -0,0 +1,204 @@
+| `variants.dark.urls.brandOverride`    | String        | URL for the `dark.css` file from `@openedx/brand-openedx`.                                                    |
+
+### JavaScript-based configuration
+
+One approach to configuring the `PARAGON_THEME_URLS` is to create a `env.config.js` file in the root of the repository. The configuration is defined as a JavaScript file, which affords consumers to use more complex data types, amongst other benefits.
+
+To use this JavaScript-based configuration approach, you may set a `PARAGON_THEME_URLS` configuration variable in a `env.config.js` file:
+
+```js
+const config = {
+    PARAGON_THEME_URLS: {
+        core: {
+            url: 'https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/core.min.css',
+        },
+        defaults: {
+            light: 'light',
+        },
+        variants: {
+            light: {
+                url: 'https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/light.min.css',
+            },
+        },
+    },
+};
+
+export default config;
+```
+
+### MFE runtime configuration API
+
+`@openedx/frontend-platform` additionally supports loading application configuration from the MFE runtime configuration API via `edx-platform`. The configuration is served by the `http://localhost:18000/api/mfe_config/v1` API endpoint. For more information, refer to [this documentation](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/mfe_config_api/docs/decisions/0001-mfe-config-api.rst) about the MFE runtime configuration API, please see these docs.
+
+The application configuration may be setup via Django settings as follows:
+
+```python
+ENABLE_MFE_CONFIG_API = True
+MFE_CONFIG = {}
+MFE_CONFIG_OVERRIDES = {
+    # The below key represented the `APP_ID` defined in your MFE
+    'profile': {
+        'PARAGON_THEME_URLS': {
+            'core': {
+                'url': 'https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/core.min.css',
+            },
+            'defaults': {
+                'light': 'light',
+            },
+            'variants': {
+                'light': {
+                    'url': 'https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/light.min.css',
+                },
+            },
+        },
+    },
+}
+```
+
+### Locally installed `@openedx/paragon`
+
+If you would like to use the same version of the Paragon CSS urls as the locally installed `@openedx/paragon`, the configuration for the Paragon CSS urls may contain a wildcard `$paragonVersion` which gets replaced with the locally installed version of `@openedx/paragon` in the consuming application, e.g.:
+
+```shell
+https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/core.min.css
+https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/light.min.css
```

Comment:
No, because the main intention here is to identify the version and load the external resource compatible with that specific version. 

For example, my app has paragon `23.0.0-alpha.2`, then the hook will replace `$paragonVersion` .

This is the definition
`https://cdn.jsdelivr.net/npm/@openedx/paragon@$paragonVersion/dist/core.min.css`

after processing 
`https://cdn.jsdelivr.net/npm/@openedx/paragon@$23.0.0-alpha.2/dist/core.min.css`

Reply:
Thanks for the explanation 💯